### ISSUE TITLE
Delete Icon

### ISSUE THREAD

**Issue by hiteshah185** [TeraServ/starfish#62]
- [x] Hazard sign on delete dialog box should be **red**
- [ ] Warning message be a dialog over the main dialog box
